recipes/nvmolkit/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  version: "0.2.0"
+
+package:
+  name: "nvmolkit"
+  version: ${{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/nvMolKit/archive/v${{ version }}.tar.gz
+  sha256: 8c0181c4dcd02aac42dc3c1b783ac677a8149fd5d5b5d25e85ba30e85bd6ec5a
+
+build:
+  number: 0
+  skip:
+    - osx
+    - win
+    - cuda_compiler == 'None'
+requirements:
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('cuda') }}
+    - cmake >=3.30
+    - make
+  host:
+    - cuda-nvtx-dev
+    - eigen
+    - libboost-devel
+    - libboost-headers
+    - libboost-python-devel
+    - libcublas-dev
+    - libcurand-dev
+    - libcusolver-dev
+    - libgomp
+    - librdkit-dev
+    - pip
+    - python
+    - rdkit ==${{ rdkit }}
+    - scikit-build
+    - setuptools
+  run:
+    - cuda-cudart
```

Comment:
Is this dynamically linking to `libcudart.so`? If so, would drop this and add `cuda-cudart-dev` in `host` above

```suggestion
```